src/MigrationLog/Admin/Migrations/index.js

```diff
@@ -290,7 +290,7 @@ const Migrations = () => {
                 return null;
             }
 
-            if (migration.status === 'failed' || migration.status === 'running') {
+            if (migration.status === 'running') {
```

Comment:
Should we display the re-run option for all failed migrations? I think it is safer to display it only for those that have the `reverse()` method implemented because it can generate unexpected outcomes in the previous migrations that were implemented without the concern about having the ability to be re-run.

Reply:
I don't think so. What if failed migration is responsible for table creation?